renderer.php

```diff
@@ -838,9 +833,9 @@ protected function post_walker($cm, $discussion, $posts, $parent, $canreply, &$c
 
                     if ($post->nestedreplycount > 2 && $depth == 0) {
                         // Adding collapsable elements
-                        $output .= "<div id=".$post->id." class='posts-collapse-container collapse'>";
+                        $output .= "<div id=id".$post->id." class='posts-collapse-container collapse'>";
```

Comment:
Did this not create an issue with opening up the collapsed posts

Reply:
without the "id{post->id}" the collapse wouldn't fire

Reply:
Happy